main.tf

```diff
@@ -74,3 +74,15 @@ resource "aws_s3_bucket" "default" {
 
   tags = module.default_label.tags
 }
+
+# Refer to the terraform documentation on s3_bucket_public_access_block at
+# https://www.terraform.io/docs/providers/aws/r/s3_bucket_public_access_block.html
+# for the nuances of the blocking options
+resource "aws_s3_bucket_public_access_block" "default" {
+  bucket = aws_s3_bucket.default.id
```

Comment:
```suggestion
  bucket = join("", aws_s3_bucket.default.*.id)
```

Because of this error:
```
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121:   on ../../main.tf line 82, in resource "aws_s3_bucket_public_access_block" "default":                                         
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121:   82:   bucket = aws_s3_bucket.default.id                                                                                      
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121: Because aws_s3_bucket.default has "count" set, its attributes must be accessed                                                 
TestExamplesComplete 2020-04-02T00:56:09Z command.go:121: on specific instances.                                                                                                         
```

Reply:
Sorry about that, I missed a commit when I rebased. All fixed now.